src/route/nodes/netid/nodeEdit/NodeEdit.tsx

```diff
@@ -121,26 +121,33 @@ export const NodeEdit: React.FC<{
             </span>
           </Tooltip>
         </Grid>
-        <Tooltip title={String(t('helper.dynamicendpoint'))} placement="top">
-          <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-            <NmFormInputSwitch
-              label={String(t('node.isstatic'))}
-              name={'isstatic'}
-              defaultValue={isIPDynamic}
-            />
-          </Grid>
-        </Tooltip>
-        <Tooltip title={String(t('helper.defaultlistenport'))} placement="top">
-          <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-            <NmFormInputText
-              defaultValue={String(node.listenport)}
-              name={'listenport'}
-              label={String(t('node.listenport'))}
-              type="number"
-              disabled={node.isserver || node.udpholepunch}
-            />
-          </Grid>
-        </Tooltip>
+        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
+          <Tooltip title={String(t('helper.dynamicendpoint'))} placement="top">
+            <span>
+              <NmFormInputSwitch
+                label={String(t('node.isstatic'))}
+                name={'isstatic'}
+                defaultValue={isIPDynamic}
+              />
+            </span>
+          </Tooltip>
+        </Grid>
+        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
+          <Tooltip
+            title={String(t('helper.defaultlistenport'))}
+            placement="top"
+          >
+            <span>
+              <NmFormInputText
+                defaultValue={String(node.listenport)}
+                name={'listenport'}
+                label={String(t('node.listenport'))}
+                type="number"
+                disabled={node.isserver || node.udpholepunch}
+              />
+            </span>
+          </Tooltip>
+        </Grid>
         <Tooltip title={String(t('helper.dynamicport'))} placement="top">
```

Comment:
you can put this as the alternative to the !network?.defaultudpholepunch, then there won't be 2 tooltips..